File: app/src/main/java/com/elishaazaria/sayboard/downloader/messages/ModelInfo.java
package com.elishaazaria.sayboard.downloader.messages;

import java.util.Locale;

public class ModelInfo {
    public final String url;
    public final String filename;
    public final Locale locale;

    public ModelInfo(String url, String filename, Locale locale) {
        this.url = url;
        this.filename = filename;
        this.locale = locale;
    }

    @Override
    public String toString() {
        return "ModelInfo{" +
                "url='" + url + '\'' +
                ", filename='" + filename + '\'' +
                ", locale=" + locale +
                '}';
    }
}

File: app/src/main/java/com/elishaazaria/sayboard/downloader/messages/ModelInfo.kt
package com.elishaazaria.sayboard.downloader.messages

import com.elishaazaria.sayboard.Constants
import java.util.*

data class ModelInfo(val url: String, val filename: String, val locale: Locale = Constants.UndefinedLocale) {
//    override fun toString(): String {
//        return "ModelInfo{" +
//                "url='" + url + '\'' +
//                ", filename='" + filename + '\'' +
//                ", locale=" + locale +
//                '}'
//    }
}